addons/medical_feedback/XEH_postInit.sqf

```diff
@@ -78,6 +78,27 @@ GVAR(bloodTickCounter) = 0;
     ["unconscious", _status] call EFUNC(common,setDisableUserInputStatus);
 }] call CBA_fnc_addPlayerEventHandler;
 
+// Update effects for featureCamera (curator, arsenal, etc)
+["featureCamera", {
+    params ["_unit", "_newCamera"];
+
+    [true] call FUNC(handleEffects);
+
+    private _volume = missionNamespace getVariable [QEGVAR(hearing,unconsciousnessVolume), VOL_UNCONSCIOUS];
+
+    if (_newCamera == "") then { // switched back to player view
+        private _status = IS_UNCONSCIOUS(_unit);
+        [!_status, _unit] call EFUNC(common,setVolume);
+
+        [QUOTE(ADDON), _volume, _status] call EFUNC(common,setHearingCapability);
+
+        ["unconscious", _status] call EFUNC(common,setDisableUserInputStatus);
+    } else {
+        [false, _unit] call EFUNC(common,setVolume);
+        [QUOTE(ADDON), _volume, false] call EFUNC(common,setHearingCapability);
```

Comment:
Why should we mute audio for players in curator cam or spectator?

Doesn't this go against the intention of line 63 above

Reply:
Shouldn't, misread the function params